### ISSUE TITLE
Using maven / gradle

### ISSUE THREAD

**Issue by Swissbite** [hvdwolf/jExifToolGUI#5]
I would consider using any build tool you like. Either maven or gradle.
Reasons: You may like to add some libraries to be used in run/build/development time without adding a large blob jar to your source.

Which libraries do I think should be used?
- [SLF4J](https://www.slf4j.org/) to use a proper logging tool instead of System.out.println
- [JUnit](https://junit.org/junit5/) as a testing framework

I'm more experienced in maven if I should help you in set up your project a maven like way.

**Comment by hvdwolf** [comment]
I prefer gradle. I have experience with that in a number of projects. 
It is on my personal wish list, but I first wanted to move a little further with my project.
I still need to complete the renaming (Extra menu) and GPS edit page. I think I drop gpano after all. It was "hot" in 2011-2012, but it is not anymore, so it seems.

**Comment by Swissbite** [comment]
I'm also okay in setting it up with gradle. But I would suggest to do it as soon as possible. It is much easier in the early stages of projects :-D

**Comment by hvdwolf** [comment]
As such I fully agree with you but as I did not drag anything external in my project yet or created a sub- or side project, it does not make a difference. Currently it would only be one or more java file extra inside the current project, which is then automatically incorporated. That's why I did not start it yet.
And I also need to do some clean-up first. Some methods are not used at anymore and I made an error with the resources in the start (As I did not grasp the concept well enough).

But having gradle or maven now before "your" external libraries are added, would be good.
